### DISCUSSION TITLE
How to Set a Transparent Background for CartesianChart in LiveCharts2?

### DISCUSSION THREAD

**Discussion by RobertKR21** [Live-Charts/LiveCharts2#1693] [Q&A]
Hi, I'm using LiveCharts2 with WinForms and would like to make the background of my CartesianChart transparent. I’ve tried setting BackColor = Color.Transparent, but it still shows as a solid color. Is there a way to achieve a fully transparent background, so that overlapping charts show through each other?

Thank you for any advice!

**Comment by ghost** [comment]
To make the background of a LiveCharts2 CartesianChart transparent in WinForms, you need to adjust the chart's background using the Chart's specific property and ensure that the BackColor of the control itself is transparent. Setting BackColor to Color.Transparent won't work directly as LiveCharts may be rendering over a non-transparent background.

Here’s how you can make the background transparent:

1. Set the BackColor of the CartesianChart to Transparent:
You need to set the Background of the CartesianChart control itself, not just the BackColor.


cartesianChart1.Background = Brushes.Transparent;

2. Set the Plot Area Background to Transparent:
Additionally, if the plot area itself has a background color, you should set it to transparent:

cartesianChart1.AxisX.Fill = Brushes.Transparent;
cartesianChart1.AxisY.Fill = Brushes.Transparent;

3. Ensure the Parent Control is Transparent:
Make sure the parent control (e.g., Panel or Form) where the CartesianChart is placed also has a transparent background:


cartesianChart1.Parent.BackColor = Color.Transparent;

4. Double-check Overlapping Charts:
If you have multiple charts overlapping, ensure all charts' backgrounds are set to transparent. You may also want to set other elements like series to have no fill, if required.

By following these steps, you should be able to make the CartesianChart background fully transparent and allow overlapping charts to show through.
